spec/nikto_upload_spec.rb

```diff
@@ -1,8 +1,8 @@
 require 'spec_helper'
+require 'ostruct'
```

Comment:
@rachkor this is not needed.

Reply:
@aapomm removed, thanks! LMK if you spot anything else.